### PR TITLE
Dp 3627 - add log file rotation

### PR DESCRIPTION
**Pull request by adrianweber**

### CODE REVIEW

File: config/global.json [hunk 1]
```diff
@@ -12,7 +12,8 @@
     },
     "logger": {
         "LEVEL": "info",
-        "PATH": "/tmp/"
+        "PATH": "/tmp/",
+        "MAX_SIZE": 134217728
```
Comment (mtomczew) [inline config/global.json]: Add comment - how much MB is that.

Comment (adrianweber) [inline config/global.json]: I cannot, this is JSON :p

Comment (mtomczew) [inline config/global.json]: xD

**Comment by mtomczew** [comment]
Merged into pre-master

**Pull request closed without merging**